### PR TITLE
Finalize docs for 4.0 release

### PR DESCRIPTION
**Pull request by aruniverse**
- update support policy to reflect 4.0 release
- cleanup nextver.md

### CODE REVIEW

File: docs/learning/api-support-policies.md [hunk 1]
```diff
@@ -84,8 +84,8 @@ Very rarely, a critical security issue may arise that cannot reasonably be addre
 | ------------- | --------------- | --------------- | --------------- | ----------------- | --------------- |
 | 1.x           | **End of life** | 2019-06-03      | 2020-05-07      | n/a               | 2020-11-01      |
 | 2.x           | **Maintenance** | 2020-05-07      | 2022-01-24      | 2022-12-31        | 2023-3-31       |
-| 3.x           | **Current**     | 2022-01-24      | **_2023-3-31_** | **_2023-9-30_**   | **_2024-3-31_** |
-| 4.x           | **Future**      | **_2023-3-31_** | -               | -                 | -               |
+| 3.x           | **Current**     | 2022-01-24      | **_2023-5-22_** | **_2023-11-22_**  | **_2024-5-22_** |
```
Comment (pmconne) [inline docs/learning/api-support-policies.md:87]: 3.x = Active, 4.x = Current ?

Comment (pmconne) [reply docs/learning/api-support-policies.md]: You changed 3.x to Maintenance. Should be Active. The definitions are above this table.

Comment (aruniverse) [reply docs/learning/api-support-policies.md]: Sorry copy and paste, its been fixed

**Review by pmconne — APPROVED** [review]

**Comment by calebmshafer** [comment]
@Mergifyio backport master

**Pull request merged**